### ISSUE TITLE
How can I save the ensemble which has best validation loss during training?

### ISSUE THREAD

**Issue by fatcatZF** [TorchEnsemble-Community/Ensemble-Pytorch#170]
How can I save the ensemble which has best validation loss during training?
